### PR TITLE
REALLY Add basket ID to payment BFF API.

### PR DESCRIPTION
**Pull request by douglashall**

### CODE REVIEW

File: ecommerce/extensions/basket/views.py [hunk 1]
```diff
@@ -520,6 +520,7 @@ def get_serialized_basket(self, context):
         """
 
         serialized_basket = {
+            'basket_id': context['basket_id'],
```
Comment (robrap) [inline ecommerce/extensions/basket/views.py:523]: Do you want to drop the earlier line, and just take it off the request here?

Untested:
```
'basket_id': request.basket.id
```

Comment (douglashall) [reply ecommerce/extensions/basket/views.py:523]: There is no request here.

**Review by robrap — APPROVED** [review]

File: ecommerce/extensions/basket/views.py (hunk 1 above)
Comment (robrap) [reply ecommerce/extensions/basket/views.py:523]: Sorry.  `self.request.basket.id`.  See voucher code below.

Comment (robrap) [reply ecommerce/extensions/basket/views.py:523]: Note: This isn't a big deal either way.  There will be clean-up much later when we only have the api.

Comment (douglashall) [reply ecommerce/extensions/basket/views.py:523]: Eh, I would probably refactor this mixin entirely if I was going to invest. This seems good enough.

**Pull request merged**